packages/firestore/src/local/persistence_transaction.ts

```diff
@@ -47,4 +50,12 @@ export abstract class PersistenceTransaction {
   raiseOnCommittedEvent(): void {
     this.onCommittedListeners.forEach(listener => listener());
   }
+
+  getStore<Key extends IDBValidKey, Value>(
```

Comment:
Putting this in PersistenceTransaction seems off. Only the IndexedDb implementation actually uses SimpleDb.

As commented elsewhere, a free function in `indexeddb_transaction.ts` that takes `PersistenceTransaction` as a type and downcasts would make this cleaner.

Reply:
Done